man/calculateJSD.Rd

```diff
@@ -38,11 +39,11 @@ will be disabled.)}
 assay to use for calculation.
 (Please use \code{assay.type} instead.)}
 
-\item{transposed}{Logical scalar, is x transposed with cells in rows?}
+\item{transposed}{Logical scalar, is x transposed with samples in rows?}
 
 \item{BPPARAM}{A
 \code{\link[BiocParallel:BiocParallelParam-class]{BiocParallelParam}}
-object specifying whether the JSD calculation should be parallelized.}
```

Comment:
Can `BPPARAM` be defined for all fns?

Reply:
Yes, BPPARAM can be defined for all. It should be clear what it does without function-specific definitions (for example here "JSD calculation" is already clear since that is what the function does)